js/samples/04.ai-apps/h.datasource-azureOpenAI/README.md

```diff
@@ -31,20 +31,15 @@ This guide will show you have the set up your custom copilot for Teams using Azu
 | [Visual Studio Code](https://code.visualstudio.com/download)                                                                  | Typescript build environments. Use the latest version.                                                                                                                                                                                                              |
 | [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) (5.3.x or greater) | Microsoft Visual Studio Code extension that creates a project scaffolding for your app. Use the latest version.                                                                                                                                                     |
-| [Git](https://git-scm.com/downloads)                                                                                          | Git is a version control system that helps you manage different versions of code within a repository.                                                                                                                                                               |
 | [Node.js](https://nodejs.org/en) (16 or 18)                                                                                   | Back-end JavaScript runtime environment. Recommended to use Node Version 16.x or 18.x, Node version >=19 is not supported. For more information, see [Node.js version compatibility table for project type](https://learn.microsoft.com/en-us/microsoftteams/platform/toolkit/build-environments#nodejs-version-compatibility-table-for-project-type)                                                                                                                                          |
-| [Microsoft Teams](https://www.microsoft.com/microsoft-teams/download-app)                                                     | Microsoft Teams to collaborate with everyone you work with through apps for chat, meetings, and call-all in one place.                                                                                                                                              |
-| [Microsoft&nbsp;Edge](https://www.microsoft.com/edge) (recommended) or [Google Chrome](https://www.google.com/chrome/)        | A browser with developer tools.                                                                                                                                                                                                                                     |
-| [Microsoft 365 developer account](/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant)                  | Access to Teams account with the appropriate permissions to install an app and [enable custom Teams apps and turn on custom app uploading](../../../concepts/build-and-test/prepare-your-o365-tenant.md#enable-custom-teams-apps-and-turn-on-custom-app-uploading). |
```

Comment:
Same question - don't they need developer accounts and azure openai?

Reply:
I was trying to simplify again. A developer account is not needed for the test tool, but it is for test in edge or to deploy the bot. I noted in the Microsoft Teams requirement the need for custom app enablement. I'll add this back in

Reply:
added back in with the latest commit